src/clnt_vc.c

```diff
@@ -266,7 +265,7 @@ clnt_vc_call(CLIENT *clnt, AUTH *auth, rpcproc_t proc,
 	SVCXPRT *xprt = &rec->xprt;
 	struct xdr_ioq *xioq;
 	XDR *xdrs;
-	rpc_ctx_t *ctx;
```

Comment:
why is it required to change the name of rpc_ctx_t?

Reply:
In general, I've been removing non-ANSI or potentially conflicting with standard typedef *_t and using the struct instead, as we discussed some time ago.

7 lines in clnt_vc.c and 5 lines in svc_vc.c is "a lot of cosmetic churn"?

Reply:
Yes, it's just churn, and removes historical data I care about, such as the role of Linux Box/CohortFS in writing these files (for example).